src/components/Tooltip/Tooltip.tsx

```diff
@@ -20,10 +20,22 @@ const Tooltip: FC<TooltipProps> = ({ tooltipData, zoom }) => {
   const { coords, disposition } = tooltipData;
   const tooltipRef = useRef<HTMLDivElement>(null);
   const width = zoom === 0 ? weekWidth : dayWidth;
-  const xOffset = width / 2 + (tooltipRef.current?.offsetWidth ?? 0) / 2;
-  const xPos = coords.x - xOffset;
+
+  useLayoutEffect(() => {
+    // re calculate tooltip width before repaint
+    if (!tooltipRef.current) return;
+
+    const { width: toolTipWidth } = tooltipRef.current.getBoundingClientRect();
+
+    const xOffset = toolTipWidth / 2 + width / 2;
```

Comment:
Maybe `tooltipWidth` (small t in Tip) as in `tooltipRef`?

Reply:
fixed in [4b5758a](https://github.com/Bitnoise/react-scheduler/pull/6/commits/4b5758a231bcd863f5149ae8dfd5be77dd67eeab)